docs/source/analysis/dataset_degree_distributions.rst

```diff
@@ -0,0 +1,6 @@
+Dataset Degree Distributions
+============================
+.. todo:: @mberr add explanation on how to interpret this plot
```

Comment:
@mberr can you please add a short explanation here of what this chart is and how to interpret it?

Reply:
ae05d22